src/site/stages/build/plugins/create-build-settings.js

```diff
@@ -31,7 +31,7 @@ function createBuildSettings(options) {
       if (!entryName) continue;
 
       const validTypes = ['string', 'boolean', 'number'];
-      const keysToSkip = new Set(['headerFooterData']);
+      const keysToSkip = new Set(['headerFooterData', 'featureToggles']);
```

Comment:
I'm not sure what this does- mostly just following the pattern I copied from `headerFooterData`

Reply:
This function creates the `settings.js` file (and `window.settings` from in the browser) using the properties found in each react app.

Without adding `'featureToggles'` to this set, it would write the feature toggles to the settings.

Reply:
Yeah, I don't think the `settings.js` file is used anymore currently, but I could see it having usefulness in the future. The point of it is that it remains human-readable and outside of our bundles, so we could manually edit the actual file on an S3 server if things are too time-sensitive for an additional build.